windows/src/engine/tsysinfox64/tsysinfox64.dproj

```diff
@@ -118,6 +118,7 @@
         <DCC_DebugDCUs>true</DCC_DebugDCUs>
         <VerInfo_Keys>CompanyName=;FileVersion=5.0.0.26;InternalName=;LegalCopyright=;LegalTrademarks=;OriginalFilename=;ProductVersion=1.0.0.0;Comments=;ProgramID=com.embarcadero.$(MSBuildProjectName);FileDescription=$(MSBuildProjectName);ProductName=$(MSBuildProjectName)</VerInfo_Keys>
         <VerInfo_Locale>1033</VerInfo_Locale>
+        <BT_BuildType>Debug</BT_BuildType>
```

Comment:
Is this change intended?

Reply:
It was added by Delphi when I saved the group project. It is harmless (we always specify the build type directly during builds).